sycl/include/sycl/detail/pi.h

```diff
@@ -199,9 +199,10 @@
 // 16.56 Replaced piextUSMEnqueueMemset with piextUSMEnqueueFill
 // 16.57 Added mappings to UR launch properties extension
 // (piextEnqueueKernelLaunchCustom)
+// 16.58 Added context parameter to piextMemImageGetInfo
 
 #define _PI_H_VERSION_MAJOR 16
```

Comment:
this should be bumped too if there is a change to `pi*` function signature.

Reply:
done